security_groups.tf

```diff
@@ -57,6 +72,38 @@ resource "exoscale_security_group_rules" "all_machines" {
   }
 }
 
+resource "exoscale_security_group" "load_balancers" {
+  name        = "${var.cluster_id}_load_balancers"
+  description = "${var.cluster_id} load balancer VMs"
+}
+resource "exoscale_security_group_rules" "load_balancers" {
+  security_group = exoscale_security_group.load_balancers.name
+  ingress {
+    description = "Kubernetes API"
+    protocol    = "TCP"
+    ports       = ["6443"]
+    cidr_list   = ["0.0.0.0/0", "::/0"]
+  }
+  ingress {
+    description = "Ingress controller TCP"
+    protocol    = "TCP"
+    ports       = ["80", "443"]
+    cidr_list   = ["0.0.0.0/0", "::/0"]
+  }
+  ingress {
+    description = "Ingress controller UDP"
+    protocol    = "UDP"
+    ports       = ["80", "443"]
+    cidr_list   = ["0.0.0.0/0", "::/0"]
+  }
```

Comment:
Why also UDP on 80/443?

Reply:
I kept this rule from the previous version of the code, but I don't think it's needed looking at the [OCP4.7 documentation](https://docs.openshift.com/container-platform/4.7/installing/installing_bare_metal/installing-bare-metal.html#installation-network-user-infra_installing-bare-metal) again.

Reply:
ok. Might be worth adding a comment to this documentation in the source?

Reply:
The documentation is linked at the top of the file